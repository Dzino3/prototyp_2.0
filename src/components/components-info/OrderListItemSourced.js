import React from 'react'

function OrderListItemSourced({item}) {
  return (
    <div className='orderListItemSourced'>
      <div className='block'>
        <p className='p2'>Factory delivery date: </p>
        <p>{item.FDD}</p>
        <p className='p2'>FDD - RDD: </p>
        <p>{item.FDD_MINUS_RDD}</p>
        <p className='p2'>Sold: </p>
        <p>{item.SOLD}</p>
      </div>
      <div className='block'>
        <p className='p2'>Order entry date: </p>
        <p>{item.ORDER_ENTRY_DATE}</p>
        <p className='p2'>Order source date: </p>
        <p>{item.ORDER_SOURCE_DATE}</p>
        <p className='p2'>Req. ship date: </p>
        <p>{item.REQ_SHIP_DATE}</p>
      </div>
      <div className='block'>
        <p className='p2'>Machine status: </p>
        <p>{item.MACHINE_STATUS}</p>
        <p className='p2'>Order status: </p>
        <p>{item.ORDER_STATUS}</p>
      </div>
      <div className='block'>
        <p className='p2'>Pres. date: </p>
        <p>{item.PRES_DATE}</p>
        <p className='p2'>Base Code Desciption: </p>
        <p>{item.BASE_CODE_DESCR}</p>
        <p className='p2'>Qty.: </p>
        <p>{item.QTY}</p>
        </div>
    </div>
  )
}

export default OrderListItemSourced
