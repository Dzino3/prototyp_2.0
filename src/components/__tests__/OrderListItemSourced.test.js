import React from "react";
import renderer from 'react-test-renderer';
import OrderListItemSourced from '../components-info/OrderListItemSourced';

describe(`Rendering OrderListItemSourced`, () => {

    test('should render OrderListItemSourced with item SOURCED', function () {
        let itemX = {
            FDD: "26.09.2022"
        };
        let tree = renderer.create(<OrderListItemSourced item={itemX} t={key => key}></OrderListItemSourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})