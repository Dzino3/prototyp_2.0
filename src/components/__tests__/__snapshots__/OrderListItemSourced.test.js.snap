// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering OrderListItemSourced should render OrderListItemSourced with item SOURCED 1`] = `
<div
  className="progressBarhelper"
>
  <div
    className="orderListItemSourced"
  >
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemSLabel1
         
      </p>
      <p>
        26.09.2022 (0 days left)
      </p>
      <p
        className="p2"
      >
        listItemSLabelRDD
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel2
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemSLabel4
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel5
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel6
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemSLabel3
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel7
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel8
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
        listItemSLabel9
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel10
         
      </p>
      <p />
      <p
        className="p2"
      >
        listItemSLabel11
         
      </p>
      <p />
    </div>
  </div>
  <div
    className="progressBarImgS"
  />
</div>
`;
